zio-http/src/main/scala/zio/http/netty/EventLoopGroups.scala

```diff
@@ -0,0 +1,67 @@
+package zio.http
+package netty
```

Comment:
This pattern makes it VERY confusing to see where things are coming from, because it implicitly imports **ALL OF** `zio.http` into scope.

Please always use:

```scala
package zio.http.netty
```

combined with explicit imports.

🙏